.github/workflows/CreatePullRequest.yml

```diff
@@ -5,40 +5,17 @@ on:
     types: ["opened"]
 
 jobs:
-  assign_and_create_card:
-    name: Assign issue to sender and create Kanban card
+  CreateCardForStandalonePR_job:
+    name: Assign PR to the author and create a Kanban card
```

Comment:
I would replace Kanban with "project" or "GitHub" project

Reply:
I find "Kanban" more clear about what it really does. While "project" and "GitHub" are technically correct, they sound ambiguous.

Reply:
> Kanban is a scheduling system for lean manufacturing Taiichi Ohno, an industrial engineer at Toyota, developed kanban to improve manufacturing efficiency.

Kanban is a process, we might follow this or another process in the future but this script is about GitHub projects and I would stick to that. I prefer to be technically correct.

Reply:
As long as that board is called Kanban, it's not wrong. And same board is called Kanban also in Jira instances.